### ISSUE TITLE
Keepmenu stopped working after a system update

### ISSUE THREAD

**Issue by Haze-sh** [firecat53/keepmenu#158]
Keepmenu was working as expected until a recent system update, I am not sure which packages caused the problem but I tried reinstalling keepmenu with all it's dependencies and it is still giving an error.

**OS**: Artix
**Keepmenu version**: keepmenu-1.4.1-1 and keepmenu-git-r272.a728d50-1

The error:

```
Traceback (most recent call last):
  File "/usr/bin/keepmenu", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.11/site-packages/keepmenu/__main__.py", line 213, in main
    if port_in_use(port) is False:
       ^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/keepmenu/__main__.py", line 36, in port_in_use
    return s.connect_ex(('localhost', port)) == 0
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
socket.gaierror: [Errno -2] Name or service not known
```

**Comment by firecat53** [comment]
I'm really going to need to know what package updated that caused the error. Can you please rollback your update and try to figure that out? I'm on NixOS unstable which should be just about as up-to-date as Arch/Artix. Thanks!

**Comment by Haze-sh** [comment]
After further investigation it turned out after the system update somehow the `localhost` can't be resolved to `127.0.0.1` and I can't find which package update made this but just changing line:5 in the main.py file do it. It will happen again after an update and it is trivial to fix now but could you consider changing it once and for all?

**Comment by firecat53** [comment]
Fixed with #156. Does this solve the original issue for you? Thanks for the interest!

**Comment by Haze-sh** [comment]
Yes this solves it. Thank you for the great software!
